### ISSUE TITLE
QTE Accessibility Option

### ISSUE THREAD

**Issue by davidsdomain** [hedge-dev/UnleashedRecomp#1131]
In recent HD games, a notable accessibility toggle that always peaks my eye makes it so any QTE that requires the repeated tapping of a single button to succeed in an action now boils down to just holding that same button down to achieve the same action, achieving the perfect mash. On top of wasting less time for casual runs _and_ speedruns, this makes it better for people with hand or arm-related disabilities that can not do such an action as easily as myself.

Also, there should be a "Hold" prompt or a different animation for the buttons to imply that you need to hold the button and not tap it. I believe that regardless of the option chosen, The Last of Us has rotating lines around the button, regardless of whether you are tapping or holding the button.

Notable and "recent" examples I can think of games that have this include The Last of Us franchise on PS5 and 2019's Star Wars Jedi: Fallen Order.

The Warehog has these mashing-style QTEs for things like activating mandatory levers, and I think it would be excellent in general for Unleashed Recomp to receive an Accessibility tab, along with something like **"Hold or Toggle to Run"** as the Werehog. 

This would be a perfect first addition to an accessibility tab that Unleashed Recomp should have. Having _another_ option to relegate it to only a single press would be even better, but one thing at a time. 

While I do not have a health issue that would make this option just for me, and this is a niche feature, for sure, I think it would be great to streamline the Werehog gameplay even more.

**Comment by DarioSamo** [comment]
> This would be a perfect first addition to an accessibility tab that Unleashed Recomp should have. Having another option to relegate it to only a single press would be even better, but one thing at a time.

Yes, while this is not in the roadmap, I'm in agreement this should be in consideration to be part of the base project at some point along with a few other options.

**Comment by davidsdomain** [comment]
> > This would be a perfect first addition to an accessibility tab that Unleashed Recomp should have. Having another option to relegate it to only a single press would be even better, but one thing at a time.
> 
> Yes, while this is not in the roadmap, I'm in agreement this should be in consideration to be part of the base project at some point along with a few other options. Probably not a priority but it's been discussed.

Thanks! Will bring up more enhancements as I play. I played Sonic Unleashed as kid, but after playing so many more games, I feel like the game industry has advanced more to include cool options like this.
